courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/write_low_level_code.ipynb

```diff
@@ -22,10 +22,8 @@
     "\n",
```

Comment:
<p>The TODO's should correspond to the objective number. It is o.k. to have multiple TODO 1's, TODO 2's, or TODO 3's.</p><p> </p><p>Same comment for remaining TODO's below.</p><p> </p><p>This is challenging lab because there are no text headers to correspond to the objectives.  Let me take this one on.  Thank you Kalyani.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1091/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Updated the TODOs with respect to the objectives of the lab. 

Thanks!